bin/crew

```diff
@@ -809,6 +848,12 @@ def prepare_package (destdir)
     # than install-info.
     # https://www.debian.org/doc/debian-policy/ch-docs.html#info-documents
     FileUtils.rm "#{CREW_DEST_PREFIX}/share/info/dir" if File.exist?("#{CREW_DEST_PREFIX}/share/info/dir")
+    
+    # Remove all perl module files which will conflict
+    if @pkg.name.include? 'perl_'
```

Comment:
I'm wondering if we should make this a regex check like `if @pkg.name ~= /^perl_/` to make sure the package name starts with perl and not some rouge package named `this_aint_perl_dude` (not that anybody would create a package with that name but just to emphasize the point).

Reply:
Sounds good.

Reply:
done